src/core/getProfits.ts

```diff
@@ -120,9 +147,29 @@ export async function getProfits({
         calculationData: {
           withdrawals: withdrawal,
           deposits: deposit,
-          startPositionValue: startPositionValue,
-          endPositionValue: endPositionValue,
+          startPositionValue: isBorrow
+            ? startPositionValue * -1
+            : startPositionValue,
+          endPositionValue: isBorrow ? endPositionValue * -1 : endPositionValue,
```

Comment:
In order to avoid repeating this logic branching, we can create a `profitModifier` and just set it up once.

```
const profitModifier =
        adapter.getProtocolDetails().positionType === PositionType.Borrow
          ? -1
          : 1
 ```
 
 Then we just always multiply by that value.